src/index.js

```diff
@@ -189,34 +193,42 @@ export default class Undo {
   }
 
   /**
-   * Sets events listeners to allow keyboard actions support.
+   * Sets up the shortcuts to undo and redo
+   * @param {string} shortcutUndo shortcut to the Undo action, it is by default ctrlKey+z
+   * @param {string} shortcutRedo shortcut to the Redo action, it is by default ctrlKey+y
+   */
+
+  shortcut(shortcutUndo = this.shortcutUndo, shortcutRedo = this.shortcutRedo) {
+    this.shortcutRedo = shortcutRedo;
+    this.shortcutUndo = shortcutUndo;
+  }
```

Comment:
@MarioRodriguezS I think we can manage this as a plugin configuration instead of requesting the user to set the shortcodes through an initialization function `shortcut`.

My two cents:
1. Add a config variable at the plugin constructor.

```javascript
constructor({ editor, onUpdate, maxLength, config }) { ... }
```

2. Set the shortcodes when initializing the plugin
```javascript
const config = {
  shortcodes: {
    undo: 'CMD+Z',
    redo: 'SHIFT+CMD+Z'
  }
}

new Undo({ editor, config });
```
What do you think?